resources/assets/uad_lists.json

```diff
@@ -2963,11 +2963,11 @@
   {
     "id": "com.motorola.launcherconfig",
     "list": "Oem",
-    "description": "Config file of the motorola launcher? \nI guess launcher will not work anymore if you delete this package. Can someone confirm?\nDON'T REMOVE THIS IF YOU DIDN'T INSTALL ANOTHER LAUNCHER!\n",
+    "description": "PIN unlock config, apparently! Delete it and you won't need a pin to unlock the launcher anymore.\nYou could still have the PIN for other admin tasks.\n",
     "dependencies": [],
     "neededBy": [],
     "labels": [],
-    "removal": "Advanced"
+    "removal": "Expert"
```

Comment:
Why the `expert` removal if your description implies no harm?

Reply:
Deleting the PIN entry is a very specific thing to want to do, so I decided that it does harm. I suppose Advanced is okay too.

Reply:
> Deleting the PIN entry is a very specific thing to want to do, so I decided that it does harm. I suppose Advanced is okay too.

It should be at least expert, because how will you unlock your phone if you've set a PIN but remove the package? Maybe we should clarify that in the description.

Reply:
That's the thing, you *can* unlock your phone *with* a PIN set!

Reply:
You just...don't need a pin(!)

Reply:
Ok, so you mean it like this: user has set a global pin, but has disabled unlocking using a PIN? But then you disable the package? Why setting a PIN if you don't need it anyway?